src/pages/AdminPage/AdminPage.tsx

```diff
@@ -268,7 +262,7 @@ const AdminPage = () => {
         },
         {
           label: 'Are they admin?',
-          render: (u: IUser) => (u.role === 0 ? 'No' : 'Yes'),
```

Comment:
maybe also add a column for Opt Out status, not just Force Opt Out?

Reply:
This inspired me to go on a whole rewording spree :0